odh/overlays/moc/monitoring/dashboards/thoth-station/thoth-service-metrics.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: integreatly.org/v1alpha1
+kind: GrafanaDashboard
+metadata:
+  name: thoth-user-api-metrics
+  labels:
+    app: grafana
+    component: thoth-station
+spec:
+  name: thoth-user-api-metrics
```

Comment:
I think (?) Grafana requires this name to contain the file type suffix (`.json`), because it translates to a filename which is then mounted to Grafana:

https://github.com/operate-first/apps/blob/master/docs/odh/grafana/add_grafana_dashboard.md#add-a-grafana-dashboard-to-operatefirst-grafana
https://github.com/integr8ly/grafana-operator/blob/master/documentation/dashboards.md

Reply:
Thanks @tumido, then I have to update all other dashboards as well. see for example: https://github.com/operate-first/apps/blob/7fac10d12109b0b53d9750bdb9c6269ce3bcc0d9/odh/overlays/moc/monitoring/dashboards/thoth-station/thoth-knowledge-graph-content-metrics.yaml#L9 and all others Thoth related: https://github.com/operate-first/apps/tree/master/odh/overlays/moc/monitoring/dashboards/thoth-station cc @harshad16

Reply:
Oh, does work just fine for the other dashboards without the suffix? Maybe I interpret it wrong and you can leave it.. :smile:

Reply:
Even though it appears weird to have requirement to put `.json` in the name, dashboards can be shared in other formats in Grafana: https://github.com/integr8ly/grafana-operator/blob/faceedabf2864402cc45767fe44ccd56e1b7955a/pkg/apis/integreatly/v1alpha1/grafanadashboard_types.go#L20

Reply:
Maybe we need to improve doc upstream to grafana operator and operate first doc consequently :D

Reply:
https://github.com/integr8ly/grafana-operator/issues/339

Reply:
Seems like this request is no longer needed, @tumido  can you resolve it?